cgc/coclustering.py

```diff
@@ -33,8 +33,17 @@ class Coclustering(object):
     """
     Perform the co-clustering analysis of a 2D array
     """
-    def __init__(self, Z, nclusters_row, nclusters_col, conv_threshold=1.e-5,
-                 max_iterations=1, nruns=1, epsilon=1.e-8, output_filename=''):
+    def __init__(self,
+                 Z,
+                 nclusters_row,
+                 nclusters_col,
+                 conv_threshold=1.e-5,
+                 max_iterations=1,
+                 nruns=1,
+                 epsilon=1.e-8,
+                 low_memory=False,
+                 numba_jit=False,
```

Comment:
```suggestion
```
@meiertgrootes Hi Meiert the `numba_jit` is the argument for `run_threads` right? Maybe we do not need to add it here.

Reply:
technically its no to run threads, but within coclustering (in the numpy version).

Reply:
Sorry, I see what you meant now. Yes, that is fine.